dashboard/dashboard-intro.md

```diff
@@ -6,6 +6,10 @@ category: how-to
 
 # TiDB Dashboard Introduction
 
+> **Warning:**
```

Comment:
This is not from https://github.com/pingcap/docs-cn/pull/3330/files. Where does this warning come from? @jackysp

Reply:
https://github.com/pingcap/docs-cn/pull/3330/files#diff-c9747b847859e83c802693a25cb125c2R1351

Reply:
The note should be in post-installation-check.md. See https://github.com/pingcap/docs/pull/2832/commits/16ce338c64886bef9c94fa2801d0e5d453db0727

Reply:
Decided to keep this note in dashboard/dashboard-intro.md. I will add this note in docs-cn later.

Reply:
Updated this note in https://github.com/pingcap/docs-cn/pull/3675